Source/WebCore/page/DOMTimer.cpp

```diff
@@ -425,30 +417,6 @@ std::optional<MonotonicTime> DOMTimer::alignedFireTime(MonotonicTime fireTime) c
     return adjustedFireTime - (adjustedFireTime % alignmentInterval) + alignmentInterval + randomizedOffset;
 }
 
-void DOMTimer::makeOpportunisticTaskDeferralScopeIfPossible(ScriptExecutionContext& context)
```

Comment:
This is still declared in DOMTimer.h but should be removed.

Reply:
Removed.